pyramus/src/main/java/fi/otavanopisto/pyramus/json/students/StudentCardServlet.java

```diff
@@ -52,6 +52,8 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response) t
       return;
     }
     
+    logger.info("Fetching student card list.");
```

Comment:
Tuleeko näitä loki täyteen, jos kyselevät kovinkin tiiviiseen tahtiin? Olisiko parempi joku singleton scheduler, jolle servlet vinkkaa viimeisimmän pyyntöhetken ja scheduler sitten lokittaisi sen (ja kenties montako kertaa sitä on ennen edellistä lokitusta kutsuttu) 1-4 kertaa päivässä?

Reply:
10min taisi olla puhe, että intervalli olisi.